src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsauditapi/model/AuditFilterDto.kt

```diff
@@ -0,0 +1,41 @@
+package uk.gov.justice.digital.hmpps.hmppsauditapi.model
+
+import io.swagger.v3.oas.annotations.media.Schema
+import javax.validation.constraints.Pattern
+import javax.validation.constraints.Size
+
+data class AuditFilterDto(
+  @Schema(
+    required = false,
+    description = "The start date and time to filter the audit events",
+    example = "2020-07-01T15:15:30Z"
+  )
+  @field:Size(min = 20, max = 20)
+  @field:Pattern(regexp = "(?i)^\\d\\d\\d\\d\\-\\d\\d\\-\\d\\d\\w\\d\\d:\\d\\d:\\d\\d\\w\$")
+  val startDateTime: String? = null,
```

Comment:
Rather than passing in a string, just pass in an Instant and Spring will do the validation for you (and you shouldn't need the pattern matcher or field size validator)

Reply:
Good point, I thought of that as well actually. Will plug that in and see how the spring validation compare.

Reply:
Done